fdbserver/Knobs.cpp

```diff
@@ -342,6 +342,11 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( PROXY_COMPUTE_BUCKETS,                                20000 );
 	init( PROXY_COMPUTE_GROWTH_RATE,                             0.01 );
 
+	init( RESET_MASTER_BATCHES,                                   200 );
```

Comment:
Can we buggy this number to cover the scenario when we have to change the value online?